### PR TITLE
Add ARM support. Migrate to Github Actions

### PR DESCRIPTION
**Pull request by stafot**
#### Summary
Add ARM support. Migrate to Github Actions

#### Ticket Link
https://mattermost.atlassian.net/browse/CLD-6838

#### Release Note

```release-note
Add ARM support. Migrate to Github Actions
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,49 +1,192 @@
 # Copyright (c) 2020-present Mattermost, Inc. All Rights Reserved.
 # See LICENSE.txt for license information.
 
+################################################################################
+##                             VERSION PARAMS                                 ##
+################################################################################
+
+## Docker Build Versions
+DOCKER_BUILD_IMAGE = golang:1.20
```
Comment (gabrieljackson) [inline Makefile:9]: One more

**Review by gabrieljackson — COMMENTED** [review]
Just have a question on versioning standardization.

File: Makefile (hunk 1 above)
Comment (stafot) [reply Makefile:9]: @gabrieljackson Do you mean to bump to 1.21?

**Review by angeloskyratzakos — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (gabrieljackson) [reply Makefile]: Sorry, lost my first comment on this. Yes I was referring to bumping to v1.21 to keep things in sync.

**Review by gabrieljackson — APPROVED** [review]

**Pull request merged**